rs/node_rewards/rewards_calculation/src/rewards_calculator_results.rs

```diff
@@ -48,6 +48,27 @@ impl DayUTC {
     pub fn unix_ts_at_day_start(&self) -> UnixTsNanos {
         (self.0.get() / NANOS_PER_DAY) * NANOS_PER_DAY
     }
+
+    pub fn next_day(&self) -> DayUTC {
+        DayUTC(DayEnd::from(self.0.get() + NANOS_PER_DAY))
```

Comment:
I'm wondering if these are really separate concepts.  Could you just have DayUTC, which can be created by giving any timestamp within a given day?